packages/genesys-spark-components/src/components/stable/gux-dropdown/gux-dropdown.scss

```diff
@@ -201,8 +201,13 @@
 
 .gux-selected-icon {
   display: flex;
+  flex-direction: row;
   align-items: center;
 
+  &.gux-icon-position-end {
+    flex-direction: row-reverse;
+  }
```

Comment:
I was just reading the accessibility for `row-reverse` (https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#accessibility) and it says it has some drawbacks for users with low vision as they wont have access to the correct  reading order as the DOM order does not change. I was wondering are we okay with this ? Would it work in the shadowDOM if you conditionally render the icon to be after the `gux-truncate` if the `iconPosition=end` ?

Reply:
I addressed this in the `gux-option-icon` component but here as the icon can only be decorative I don't think there is an accessibility concern.